src/invidious.cr

```diff
@@ -2753,14 +2753,15 @@ get "/feed/playlist/:plid" do |env|
   client = make_client(YT_URL)
   response = client.get("/feeds/videos.xml?playlist_id=#{plid}")
   document = XML.parse(response.body)
+  query_string = env.params.query.map { |s| "&#{s}" }.join
```

Comment:
I don't think this does quite what you expect.

`.map` in this case should accept two arguments: `k` and `v`. Since `env.params.query` is a `Hash(String, Array(String))`, `s` in this case is actually both arguments combined into a tuple. The above code generates:
```crystal
"&{\"param1\", \"value1\"}&{\"param2\", \"value2\"}"
```

Calling `env.params.query.to_s` directly (or preferably as `&#{env.params.query}` when building `node["url"]` below) should produce the result you want.